open_instruct/ground_truth_utils.py

```diff
@@ -600,7 +601,9 @@ def parse_completion(self, completion):
             return reasoning, score
 
         try:
-            content = completion.choices[0].message.content
+            # remove anything between <think> and </think> including the tags using regex
+            pattern = r"<think>\s*.*?\s*</think>\s*"
```

Comment:
Some models aslo wrap their answer in `<answer>...</answer>`. Lets also remove those?
```
content = content.replace("<answer>", "").replace("</answer>", "")
```